src/plugins/intel_gpu/src/kernel_selector/kernels/strided_slice/strided_slice_kernel_ref.cpp

```diff
@@ -211,7 +222,9 @@ KernelsData StridedSliceKernelRef::GetKernelsData(const Params& params, const op
 
     if (!newParams.fused_ops.empty()) {
         std::vector<std::string> idx_order;
-        if (input.Dimentions() == 5) {
+        if (input.Dimentions() == 6) {
+            idx_order = {"b", "f", "w", "z", "y", "x"};
```

Comment:
[random spot] Please add a functional test for 5d strided slice too.

Reply:
Applied